crates/node/src/cli.rs

```diff
@@ -51,6 +51,9 @@ pub struct ExtendedRunCmd {
 
     #[clap(long)]
     pub da_layer: Option<DaLayer>,
+
+    #[clap(long, short = 'm')]
+    pub l1_messages: Option<PathBuf>,
```

Comment:
`l1_messages_worker_config_file` isn't it?
Remove the `short` also

Reply:
You can use https://docs.rs/clap/latest/clap/enum.ValueHint.html with ValueHint::FilePath

Reply:
Also, I would rather have the possibility to either:
- give a path to a config file
- give two arguments `l1_contract_address` and `provider_url`

Both should be mutually exclusive. You can achieve this with clap args groups, as I did in  crates/node/src/commands/setup.rs with `struct SetupSource` using `conflicts_with`